scripts/linters/general_purpose_linter.py

```diff
@@ -33,7 +33,7 @@
     'integrations/*', 'integrations_dev/*', '*.svg', '*.gif', '*.png',
     '*.webp', '*.zip', '*.ico', '*.jpg', '*.min.js', 'backend_prod_files/*',
     'assets/scripts/*', 'core/domain/proto/*.py', 'core/tests/data/*',
-    'core/tests/build_sources/*', '*.mp3', '*.mp4', 'node_modules/*',
+    'core/tests/build_sources/*', '*.flac', '*.mp3', '*.mp4', 'node_modules/*',
```

Comment:
FWICT `flac` format is relatively larger than other audio formats. Please check with the speed team if the asset size is reasonable.

/cc @vojtechjelinek

Reply:
Replaced it with an mp3 file.